### PR TITLE
Persist activities to a file

### PR DESCRIPTION
**Pull request by 0x6431346e**
## What changed?
Activities will now persist when orion will quit.

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -35,6 +36,7 @@ import ActivitiesWindow from './windows/Activities/window'
 app.mainWindow = null
 
 // activities window
+const activityLogPath = pathJoin(app.getPath('userData'), 'activity.log')
```
Comment (koalalorenzo) [inline app/index.js]: `.json`?

**Comment by 0x6431346e** [comment]
Hmm, is it possible to resume them?

**Comment by koalalorenzo** [comment]
I don't think so. :( For now at least.

We need a different UI/structure to report failures, allow retry etc etc etc
The fact is that those actions could fail when resuming them: if the file is not there anymore we need to take care of that.

**Comment by 0x6431346e** [comment]
Alright, let's mark them as "interrupted" then :)

**Comment by koalalorenzo** [comment]
We have a bug, while I was just adding a big file to test it, it did not appear on the list of the activities. but when it was completed it was there.

https://sentry.io/siderus-opensource/orion/issues/635560601/?environment=development

**Pull request merged**